lxd-agent/api_1.0.go

```diff
@@ -1,17 +1,22 @@
 package main
 
 import (
+	"encoding/json"
+	"io/ioutil"
 	"net/http"
 	"os"
 
+	"github.com/lxc/lxd/client"
 	"github.com/lxc/lxd/lxd/response"
+	"github.com/lxc/lxd/lxd/vsock"
 	lxdshared "github.com/lxc/lxd/shared"
 	"github.com/lxc/lxd/shared/api"
 	"github.com/lxc/lxd/shared/version"
 )
 
 var api10Cmd = APIEndpoint{
 	Get: APIEndpointAction{Handler: api10Get},
+	Put: APIEndpointAction{Handler: api10Put},
```

Comment:
@monstermunchkin @stgraber Please can you remind me how we are verifying these requests from LXD's cert and not one of the other instance certs?

Reply:
Thanks @stgraber